libnmstate/validator.py

```diff
@@ -50,21 +50,19 @@ def verify_interface_capabilities(ifaces_state, capabilities):
                 "and started")
 
 
-def verify_interfaces_state(ifaces_state, ifaces_desired_state):
-    verify_link_aggregation_state(ifaces_state, ifaces_desired_state)
+def verify_interfaces_state(desired_state, current_state):
```

Comment:
Maybe this is also a good opportunity to rename it to `validate_...` - it is irritating that it is called verify but does not the usual verification.

Reply:
strictly speaking: This method is not covered by the unit tests AFAICS because the unit tests only cover the one function that this function calls. For KISS it would make sense to drop this function IMHO.

Reply:
Agreed, changing to `validate`.

This function is an abstract layer, supposed to handle interfaces state verification in general.
At the moment we have only one, and it makes sense to name that code based on what it does (link aggregation validation).
I think it is fine as it is now, but if you can think of a quick additional validation, maybe we can just add it and make it more clear.

Reply:
IMHO we can add the abstraction when we actually need it/when we have multiple things to validate. (not a must)

Reply:
I think we can add the dhcp validation, will check this later on.